src/main/java/com/github/jasminb/jsonapi/ResourceConverter.java

```diff
@@ -394,7 +394,7 @@ public void setTypeResolver(RelationshipResolver resolver, Class<?> type) {
 					if (object != null) {
 						result.add(new Resource(createIdentifier(jsonNode), object));
 					}
-				} else {
+				} else if (!deserializationFeatures.contains(DeserializationFeature.ALLOW_UNKNOWN_INCLUSIONS)) {
 					throw new IllegalArgumentException("Included section contains unknown resource type: " +
 							String.valueOf(true));
```

Comment:
You might as well fix the problem with using `true` instead of actual type name in the exception.